config/features.rb

```diff
@@ -30,7 +30,7 @@
 
   feature :batch_upload,
           default: false,
-          description: "Enable uploading batches of works"
+          description: "Enable uploading batches of works. <br /><strong>WARNING:</strong> Do not enable. This feature is broken and will be addressed in https://github.com/samvera/hyrax/issues/7185.".html_safe
```

Comment:
I already merged this but should we wrap the url in <a>? so it's linked? It looks like this deployed as is: 

<img width="1188" height="83" alt="Image" src="https://github.com/user-attachments/assets/6e029ea0-a1f3-40a6-a727-ee32bc5bfbec" />

Reply:
I updated it to 

```
  feature :batch_upload,
          default: false,
          description: "Enable uploading batches of works. <br /><strong>WARNING:</strong> Do not enable. This feature is broken and will be addressed in <a href='https://github.com/samvera/hyrax/issues/7185' target='_blank'>https://github.com/samvera/hyrax/issues/7185</a>.".html_safe
```